Automattic/jetpack
# Duplicate views: Add util to fetch experiment assignment

## Proposed changes:

Adds a new util so we can fetch the current user assignment of the Duplicate Views experiment.

Note that the experiment has not been created, so in the meantime we're using a placeholder (a temporary experiment I created that will be removed once the actual experiment has been created).

22124-explat-experiment

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
- Apply these changes to your sandbox and to your Atomic site
- Open a shell and run `wpcom_is_duplicate_views_experiment_enabled()`
- Make sure it returns whether the experiment is enabled for you or not
- You can change your assignment in ExPlat, using the bookmarklets provided by the experiment page: 22124-explat-experiment
